pysat/tests/test_meta.py

```diff
@@ -989,4 +963,15 @@ def test_transfer_attributes_to_instrument_strict_names(self):
         self.meta.transfer_attributes_to_instrument(self.testInst,
                                                     strict_names=True)
 
-        assert True
+    def test_merge_meta(self):
+        self.meta['new'] = {'units': 'hey', 'long_name': 'boo'}
+        meta2 = pysat.Meta()
+        meta2['NEW21'] = {'units': 'hey2', 'long_name': 'boo2',
```

Comment:
:)

Reply:
The meta tests are the most fun to update